### ISSUE TITLE
Is this project still being worked on?

### ISSUE THREAD

**Issue by Cursewords** [cSploit/android#673]
I know people have lives and jobs away from developing but I haven't seen a pull or an update on this project for a long time. Is there any information available from a dev to share?

**Comment by DeveloppSoft** [comment]
Working on cSploit 2 :smile: check the website :smirk:

**Comment by tux-mind** [comment]
Yep, as now i'm coding the packet analyzer ;)

**Comment by evertking** [comment]
Not a ETA.. just kinda asking.. how things are going? Love your work and been here since the first release.
I'm not expert by any means but... seems to me after Kit Kat stuff went down hill, am I right? Did it have to do with the new run time that Android uses in lollipop and forward? Again, thanks for all the work.

**Comment by tux-mind** [comment]
I'm really sorry about cSploit, I'm very busy in these days.

I've developed the daemon until few days ago, I've made the packet analyzer
and I was working on the dissector infrastructure.

I've been engaged by a IT security company for working 3 days/week + my
university projects ( 3 as now ) + the incoming exams.

I'm very sad about this delay, but believe me that I'm still there for you.

Anyway the guy who make the client has finished it's first part, so I think
that it will not take so much for a preview of cSploit 2!

As now only network analysis works ( scanning of alive hosts and ports ).

As usually your help matters, especially if you're a developer and have
some free time!

Best regard, yours, tux_mind.


2016-06-16 13:41 GMT+02:00 evertking <notifications@github.com>:
